twoliter/embedded/Makefile.toml

```diff
@@ -24,6 +24,11 @@ BUILDSYS_CARGO_METADATA_PATH = "${BUILDSYS_METADATA_DIR}/cargo_metadata.json"
 BUILDSYS_SBKEYS_PROFILE = { script = ['echo "${BUILDSYS_SBKEYS_PROFILE:-local}"'] }
 BUILDSYS_TIMESTAMP = { script = ["date +%s"] }
 BUILDSYS_VERSION_BUILD = { script = ["git describe --always --dirty --exclude '*' || echo 00000000"] }
+# The unix timestamp in ms of the latest commit of the project.
+# This is an input for setting the Release value of a package.
+BUILDSYS_VERSION_BUILD_TIMESTAMP = { script = ["git show -s --format=%ct HEAD || echo 0000000000"] }
+# The epoch field is used for application inventory generation in rpm2img.
```

Comment:
nit: I'd be more specific here.

```
The epoch field sets the RPM epoch for all built packages. This versioning information is ultimately reflected in the application inventory generated by rpm2img
```

Transparency here could make it easier to debug issues that may occur in the future.